andela/codango
# #115594687 Add in Test Coverage and Code Quality Badges on Repo README

#### What does this PR do?
* Adds Circle CI build status badge on repo README
* Adds Coveralls test coverage percentage badge on repo README
* Adds Scrutinizer code quality score badge on repo README

#### Description of tasks to be completed
Display build status, test coverage and code quality score badges
 [![Circle CI](https://circleci.com/gh/andela/codango/tree/develop.svg?style=svg)](https://circleci.com/gh/andela/codango/tree/develop) 
[![Coverage Status](https://coveralls.io/repos/github/andela/codango/badge.svg?branch=ch-add-badges-115594687)](https://coveralls.io/github/andela/codango?branch=ch-add-badges-115594687)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/andela/codango/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/andela/codango/?branch=develop)

#### How should this be manually tested?
Previewing the markdown below should show the build status, test coverage and code quality badges.
```
[![Circle CI](https://circleci.com/gh/andela/codango/tree/develop.svg?style=svg)](https://circleci.com/gh/andela/codango/tree/develop) 
```
```
[![Coverage Status](https://coveralls.io/repos/github/andela/codango/badge.svg?branch=ch-add-badges-115594687)](https://coveralls.io/github/andela/codango?branch=ch-add-badges-115594687)
```
```
[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/andela/codango/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/andela/codango/?branch=develop)
```

#### What are the relevant pivotal tracker stories?
 [#115594687] Add in Test Coverage and Code Quality Badges on Repo README
